pkg/purl/purl.go

```diff
@@ -120,12 +145,39 @@ func (p *PackageURL) LangType() ftypes.LangType {
 		return ftypes.Pub
 	case packageurl.TypeBitnami:
 		return ftypes.Bitnami
+	case TypeK8s:
+		switch p.Namespace {
+		case NamespaceEKS:
+			return ftypes.EKS
+		case NamespaceGKE:
+			return ftypes.GKE
+		case NamespaceAKS:
+			return ftypes.AKS
+		case NamespaceRKE:
+			return ftypes.RKE
+		case NamespaceOCP:
+			return ftypes.OCP
+		case "":
+			return ftypes.Kubernetes
```

Comment:
`ftypes.Kubernetes` is `ConfigType`. Or am i missing something?
```suggestion
			return ftypes.K8sUpstream
```

Reply:
You're correct.